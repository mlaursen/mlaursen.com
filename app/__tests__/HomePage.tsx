import { describe, expect, it } from "@jest/globals";
import { rmdRender } from "@react-md/core/test-utils";
import Page from "../page.js";

describe("HomePage", () => {
  it("should render correctly", () => {
    const { container } = rmdRender(<Page />);

    expect(container).toMatchSnapshot();
  });
});
