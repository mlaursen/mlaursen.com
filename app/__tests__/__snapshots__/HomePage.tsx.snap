// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage should render correctly 1`] = `
<div>
  <h1
    class="rmd-typography rmd-typography--headline-2"
  >
    Hello,
  </h1>
  <p
    class="rmd-typography rmd-typography--headline-4"
  >
    This page is empty since I haven't really made this site yet. You can check out my 
    <a
      class="rmd-link"
      href="https://github.com/mlaursen"
    >
      GitHub
    </a>
     
    for some more information about projects I work on.
  </p>
</div>
`;
